src/StagesEditor/TabWidget/ExpandingTabBar.h

```diff
@@ -12,9 +12,11 @@ class ExpandingTabBar: public QTabBar {
 	protected:
 	bool event(QEvent *event) override;
 	void paintEvent(QPaintEvent *paintEvent) override;
-	void restorePen(QPainter* painter, QPen pen, int index);
-	void setBrush(QPainter* painter, int index);
+	void restorePen(QPainter *painter, QPen pen, int index);
+	void setBrush(QPainter *painter, int index);
```

Comment:
Clean code principle: `Be consistent. If you do something a certain way, do all similar things in the same way.`